test/logger.test.js

```diff
@@ -73,5 +73,88 @@ module.exports = () => {
       expect(databuff[0]).to.be.equal('some data 3')
       expect(databuff[1]).to.be.equal('some data 4')
     }).timeout(1200000)
+
+    it('filter logs by date', async () => {
+      const databuff = []
+
+      const server = new HyperCoreLogger(() => ram())
+      const push = (date) => server.feed.append(date + ' some data ')
+
+      await server.start()
+      push('1970-01-01T00:00:00.000Z')
+      push('1970-01-01T00:10:00.000Z')
+      push('1970-01-01T00:20:00.000Z')
+      push('1970-01-01T00:30:00.000Z')
+      push('1970-01-01T00:40:00.000Z')
+      push('1970-01-01T00:50:00.000Z')
+
+      const client = new HyperCoreLogReader(
+        () => ram(), server.feedKey, null, null, { startDate: new Date('1970-01-01T00:05:00.000Z'), endDate: new Date('1970-01-01T00:35:00.000Z') }
```

Comment:
please add two more 3 more tests:
- test where only startDate option is present
- test where only endDate option is present
- check when start/end dates are present but start date is after last log